docs/documentation/admin/workflows/admin_workflow_configure.md

```diff
@@ -0,0 +1,519 @@
+<div className="col col--3"><b>State:</b></div>
+<div className="col col--5">Enter the name of the succeeding state.</div>
+<div className="col col--4"><em>States must have been previously saved into the workflow.</em></div>
+</div>
+<div className="row table-row-1">
+<div className="col col--3"><b>Can change:</b></div>
+<div className="col col--5">
+
+Options are: _survey_, _task-ui_ (task), or _*_ (both).
+
+</div>
+<div className="col col--4"><em></em></div>
+</div>
+<div className="row table-row-2">
+<div className="col col--3"><b>Start form:</b></div>
+<div className="col col--5">Indicates that a survey that will initialize with the task.</div>
+<div className="col col--4"><em>All created surveys will appear as options, but only the survey created for the task should be chosen.</em></div>
+</div>
+<div className="row table-row-1">
+<div className="col col--3"><b>+ Edit Routine:</b></div>
+<div className="col col--5">Routines automate processes within the workflow.</div>
+<div className="col col--4"><em>
+
+[Click here](/docs/documentation/automation/admin_routine) to learn more about routines.
+
+</em></div>
+</div>
+<div className="row table-row-title">
+<div className="col col--12"><b>Subtask</b></div>
+</div>
+<div className="row table-row-1">
+<div className="col col--3"><b>Workflow:</b></div>
+<div className="col col--5">Indicates a subtask that continues through a different workflow.</div>
+<div className="col col--4"><em>Only existing workflows in the workflow group can be selected.</em></div>
+</div>
+<div className="row table-row-2">
+<div className="col col--3"><b>+ Edit Routine:</b></div>
+<div className="col col--5">Routines automate processes within the workflow.</div>
+<div className="col col--4"><em>
+
+[Click here](/docs/documentation/automation/admin_routine) to learn more about routines.
+
+</em></div>
+</div>
+<div className="row table-row-title">
+<div className="col col--12"><b>Survey triggers</b></div>
+</div>
+<div className="row table-row-1">
+<div className="col col--3"><b>+ Add Survey Trigger:</b></div>
+<div className="col col--5">A survey is used to trigger the passage of the state to its successors. Multiple survey triggers can be added.</div>
+<div className="col col--4"><em>When pressed, new fields appear.</em></div>
+</div>
+<div className="row table-row-2">
+<div className="col col--3"><b>Survey:</b></div>
+<div className="col col--5">Choose from existing surveys.</div>
+<div className="col col--4"><em>
+
+[Click here](/docs/documentation/automation/admin_routine) to learn more about routines.
+
+</em></div>
+</div>
+<div className="row table-row-1">
+<div className="col col--3"><b>+ Edit Routine:</b></div>
+<div className="col col--5">Routines automate processes within the workflow.</div>
+<div className="col col--4"><em>
+
+[Click here](/docs/documentation/automation/admin_routine) to learn more about routines.
+
+</em></div>
+</div>
+</div>
+<br/>
+
+### Create/Edit SLA {#create-edit-sla}
```

Comment:
Lets move this to its own section. 
SLAs are very important - people always ask about this (but doesn't use it as much)

Reply:
Perhaps move this file into Automation section, and Just leave a short text in the workflows explaining SLAs - for example
```
SLAs allow us to set up an Automation if a Task stays too long a certain state .
e.g,  An critical ticket is not solved within 24 hours an email to the Manager is sent

Check the SLA section to setup Automation
``` 

![image](https://user-images.githubusercontent.com/3951895/114910855-60fd7c80-9dec-11eb-9a01-6819f67b01b1.png)

Reply:
Great idea!

Reply:
In this case, should we also send the "Required Surveys" to "Automation"?